Makefile

```diff
@@ -0,0 +1,22 @@
+default:
+	cat Makefile
+
+synthetic_data.csv:
+	wget synthetic_data.csv https://gist.github.com/Niarfe/25100a860cbb660150a3fec87378d8ac
+
+square_wave.png:
+	gnuplot -e "set datafile separator ','; set term png; set outp
+t '<output>.png'; plot 'synthetic_data.csv' using 1:2 with lines"
+
+fibonachi.png:
+	gnuplot -e "set datafile separator ','; set term png; set output
+t '<output>.png'; plot 'synthetic_data.csv' using 1:3 with lines"
+
```

Comment:
`rm` is totally the right command here, usually we only need the -f (force) flag when we want to remove a directory.  `rm` will complain if the directory is not empty, thus the need for the -f.  If you find you actually need it let me know, maybe something is wrong with permissions, but for now, let's try removing it and see if it all works.